recipe/data.go

```diff
@@ -72,44 +53,22 @@ var CPIH = Response{
 	},
 }
 
-var ASHEP01 = Response{
-	ID:     "7b160747-bc5a-4baa-857a-aa72545d9899",
-	Alias:  "ASHEP01",
+var MIDYEARPOPEST = Response{
+	ID:     "40AA070E-7A43-4EC5-A1FC-84CEA2BC4461",
+	Alias:  "MIDYEARPOPEST",
 	Format: "v4",
 	InputFiles: []file{
-		{"ASHEP01 v4"},
+		{"MIDYEARPOPEST v4"},
 	},
 	OutputInstances: []instance{
 		{
-			DatasetID: "ASHEP01",
+			DatasetID: "midyearpopest",
 			Editions:  []string{"Time-series"},
-			Title:     "CPIH",
-			CodeLists: []CodeList{{ID: "time", Name: "time", HRef: "http://localhost:22400/code-lists/time"},
-				{ID: "UK-only", Name: "geography", HRef: "http://localhost:22400/code-lists/UK-only"},
-				{ID: "asheagegroup", Name: "agegroup", HRef: "http://localhost:22400/code-lists/asheagegroup"},
-				{ID: "asheearnings", Name: "earnings", HRef: "http://localhost:22400/code-lists/asheearnings"},
-				{ID: "ashegender", Name: "gender", HRef: "http://localhost:22400/code-lists/ashegender"},
-				{ID: "asheprovision", Name: "provision", HRef: "http://localhost:22400/code-lists/asheprovision"},
-				{ID: "asheunits", Name: "units", HRef: "http://localhost:22400/code-lists/asheprovision"}},
-		},
-	},
-}
-
-var POPTC = Response{
-	ID:     "e295af68-d9e4-43ba-b6ea-972301229da8",
-	Alias:  "POPTC",
-	Format: "v4",
-	InputFiles: []file{
-		{"POPTC v4"},
-	},
-	OutputInstances: []instance{
-		{
-			DatasetID: "POPTC",
-			Editions:  []string{"2011"},
-			Title:     "POPTC (experimental)",
+			Title:     "Mid Year Population Estimates",
 			CodeLists: []CodeList{{ID: "time", Name: "time", HRef: "http://localhost:22400/code-lists/time"},
-				{ID: "towncities", Name: "geography", HRef: "http://localhost:22400/code-lists/townscities"},
-				{ID: "townscitiesagegroup", Name: "agegroup", HRef: "http://localhost:22400/code-lists/townscitiesagegroup"}},
+				{ID: "midyearpop", Name: "geography", HRef: "http://localhost:22400/code-lists/midyearpop"},
```

Comment:
should this be "midyearpopgeography" to be consistent?

Reply:
think I was just copying from the towns and cities one. I can change it and it'd make more sense but its back-end only but would need to alter the codelists too. Is it worth it? I'm thinking all three have different naming conventions at the moment anyway, we've got a standardising exercise to come regardless